### DISCUSSION TITLE
what potentiometers are used?

### DISCUSSION THREAD

**Discussion by tiretrak** [marcel-licence/esp32_basic_synth#38] [Q&A]
what potentiometers are used?
1k or 10k
line or log?

**Comment by marcel-licence** [comment]
I am actually using 100K linear potentiometers. Others would also work. I think linear is better to get the same resolution on both ends when turning. Using 1k might increase you current draw (U/R = I). For less current higher values would be better. For less interference maybe lower values are better.

**Reply by tiretrak** [reply]
> 
> 
> I am actually using 100K linear potentiometers. Others would also work. I think linear is better to get the same resolution on both ends when turning. Using 1k might increase you current draw (U/R = I). For less current higher values would be better. For less interference maybe lower values are better.

Thank you . I've tried using 10k. this is the default value for arduino. but the adjustment took place at the end of the value

**Reply by marcel-licence** [reply]
I did choose 100k just because it was the cheapest. I would expect that both will behave the same. The adc is just measuring the voltage which will be between VCC and GND. Maybe there is some additional experience regarding different results by using different resistors.
The automatic adjustment in the code is just because I tried using different oversampling resulting in different values by adding all measurements.
